sklearn/utils/extmath.py

```diff
@@ -682,7 +730,14 @@ def _randomized_eigsh(
     """
     if selection == "value":  # pragma: no cover
         # to do : an algorithm can be found in the Halko et al reference
-        raise NotImplementedError()
+        eigvals, eigvecs=eigen_decomposition_one_pass(
+            M,
+            n_components=n_components,
+            n_oversamples=n_oversamples,
```

Comment:
Consistency: 

- randomized svd was coded to return decreasing magnitude eigenvals to randomized eigsh module was coded the same way
- however all other eigs solvers (arpack, dense) behave the other way and return by increasing order of magnitude (to check) AND kernel PCA sorts again in the reverse order
- so eigen_decomposition_one_pass was implemented in increasing order

TODO decide. Most probably the best thing is to reverse order of the "module" implementation of randomized_eigsh so that everything is consistent. BUT the docstring should be very explicit about it.